plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -996,6 +999,61 @@ public void testAllowColumnDefaults()
         assertQueryFails("INSERT INTO allow_column_defaults (a) VALUES (2)", "\\QUnsupported writer features: [allowColumnDefaults]");
     }
 
```

Comment:
Is altering table to add/remove this property supported ?
Can we add a test for that ?

Reply:
It's unsupported for now. Added a negative test.